application/src/main/java/org/thingsboard/server/service/edge/rpc/EdgeGrpcService.java

```diff
@@ -85,11 +86,12 @@
 @TbCoreComponent
 public class EdgeGrpcService extends EdgeRpcServiceGrpc.EdgeRpcServiceImplBase implements EdgeRpcService {
 
-    private final ConcurrentMap<EdgeId, EdgeGrpcSession> sessions = new ConcurrentHashMap<>();
+    private final ConcurrentMap<EdgeId, AbstractEdgeGrpcSession> sessions = new ConcurrentHashMap<>();
```

Comment:
Either get rid of EdgeGrpcSession interface and rename AbstractEdgeGrpcSession to EdgeGrpcSession, or use interface instead of abstract class